src/component/elements/dropDownButton/DropDownList.tsx

```diff
@@ -58,16 +58,20 @@ function DropDownList({
   labelKey,
 }: InnerDropDownListProps<DropDownListItem>) {
   const ref = useRef<HTMLDivElement>(null);
+  const containerBoundariesRef = useRef<DOMRect>();
```

Comment:
Why are you caching those values because it is never reused ?

Reply:
in the development mode because of the strict mode it's triggered twice

Reply:
Because otherwise it does not work in development mode otherwise ?

Reply:
There is specific information in the documentation if there is a problem:

https://beta.reactjs.org/learn/synchronizing-with-effects#how-to-handle-the-effect-firing-twice-in-development

You may have to add a cleanup callback.

Reply:
> Because otherwise it does not work in development mode otherwise ?

yes exactly

Reply:
i think we do not need a clean-up callback because the next getBoundingClientRect() will take into account the `left` value that we add

Reply:
Your cleanup callback should undo what you do in your effect. In this case, reset the value of `ref.current.style.left`. Then when it runs a second time, it will work correctly.

Reply:
Yes, you are right, I was thinking instead of resetting the value, I can cache it.